### PR TITLE
Update template

### PR DESCRIPTION
**Pull request by the-cleaner-66**
Newer "distfiles" (date: 07.09.2020) + modified "LocalConfig.kmk" (enable virtio)

### CODE REVIEW

File: srcpkgs/virtualbox-ose/template [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+revision=2
 wrksrc="VirtualBox-${version%*a}"
 short_desc="General-purpose full virtualizer for x86 hardware"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-only, CDDL-1.0"
 homepage="https://www.virtualbox.org"
 changelog="https://www.virtualbox.org/wiki/Changelog"
-distfiles="http://download.virtualbox.org/virtualbox/${version%*a}/VirtualBox-$version.tar.bz2"
-checksum=91fa05bcfce36316ca93e3927c9550ea66286fff4c5bec900b753fca278ce1a0
+distfiles="http://download.virtualbox.org/virtualbox/${version%*a}/VirtualBox-6.1.14a.tar.bz2"
```
Comment (Johnnynator) [inline srcpkgs/virtualbox-ose/template:11]: Stay with $version and just append the `a` to the version string. And don't bump the revision.

**Review by Johnnynator — CHANGES_REQUESTED** [review]
Use commit naming as outlined in https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md

File: srcpkgs/virtualbox-ose/template [hunk 2]
```diff
@@ -1,15 +1,15 @@
 # Template file for 'virtualbox-ose'
 pkgname=virtualbox-ose
 version=6.1.14
-revision=1
+revision=2
```
Comment (the-cleaner-66) [inline srcpkgs/virtualbox-ose/template:4]: revision=1

File: srcpkgs/virtualbox-ose/template (hunk 1 above)
Comment (the-cleaner-66) [reply srcpkgs/virtualbox-ose/template:11]: distfiles="http://download.virtualbox.org/virtualbox/${version%*a}/VirtualBox-$versiona.tar.bz2"

Comment (fosslinux) [reply srcpkgs/virtualbox-ose/template:11]: `distfiles="http://download.virtualbox.org/virtualbox/${version%*a}/VirtualBox-${version}a.tar.bz2"` to make it clearer, imo

**Comment by ericonr** [comment]
This PR should include the changes from #24838

**Comment by the-cleaner-66** [comment]
Thank you for the hints!
I have one question: Where do I have to place the [ci skip] flag?
Building the package took almost 60 minutes on my workstation laptop.

Kind regards
Michael Krahl

Am Fri, 11 Sep 2020 15:01:45 -0700
schrieb John Zimmermann <notifications@github.com>:

> @Johnnynator requested changes on this pull request.
> 
> Use commit naming as outlined in
> https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md
> 
> >  wrksrc="VirtualBox-${version%*a}"
>  short_desc="General-purpose full virtualizer for x86 hardware"
>  maintainer="Orphaned <orphan@voidlinux.org>"
>  license="GPL-2.0-only, CDDL-1.0"
>  homepage="https://www.virtualbox.org"
>  changelog="https://www.virtualbox.org/wiki/Changelog"
> -distfiles="http://download.virtualbox.org/virtualbox/${version%*a}/VirtualBox-$version.tar.bz2"
> -checksum=91fa05bcfce36316ca93e3927c9550ea66286fff4c5bec900b753fca278ce1a0
> +distfiles="http://download.virtualbox.org/virtualbox/${version%*a}/VirtualBox-6.1.14a.tar.bz2"
> 
> Stay with $version and just append the `a` to the version string. And
> don't bump the revision.
> 


-- 
Michael Krahl
Weiherstraße 16
D-89257 Illertissen
Tel.: 07303-43726

**Comment by the-cleaner-66** [comment]
Perfect, thanks for your help!

Kind regards
Michael Krahl

Am Fri, 11 Sep 2020 23:49:59 +0000 (UTC)
schrieb fosslinux <notifications@github.com>:

> @fosslinux commented on this pull request.
> 
> 
> 
> >  wrksrc="VirtualBox-${version%*a}"
>  short_desc="General-purpose full virtualizer for x86 hardware"
>  maintainer="Orphaned <orphan@voidlinux.org>"
>  license="GPL-2.0-only, CDDL-1.0"
>  homepage="https://www.virtualbox.org"
>  changelog="https://www.virtualbox.org/wiki/Changelog"
> -distfiles="http://download.virtualbox.org/virtualbox/${version%*a}/VirtualBox-$version.tar.bz2"
> -checksum=91fa05bcfce36316ca93e3927c9550ea66286fff4c5bec900b753fca278ce1a0
> +distfiles="http://download.virtualbox.org/virtualbox/${version%*a}/VirtualBox-6.1.14a.tar.bz2"
> 
> `distfiles="http://download.virtualbox.org/virtualbox/${version%*a}/VirtualBox-${version}a.tar.bz2"`
> to make it clearer, imo
> 


-- 
Michael Krahl
Weiherstraße 16
D-89257 Illertissen
Tel.: 07303-43726

File: srcpkgs/virtualbox-ose/template (hunk 1 above)
Comment (the-cleaner-66) [inline srcpkgs/virtualbox-ose/template:11]: distfiles="http://download.virtualbox.org/virtualbox/${version%*a}/VirtualBox-${version}a.tar.bz2"

**Comment by pullmoll** [comment]
Working on it.

**Pull request closed without merging**